### ISSUE TITLE
Can't set authors

### ISSUE THREAD

**Issue by sebrojas14** [Automattic/Co-Authors-Plus#673]
![image](https://user-images.githubusercontent.com/1771574/56856205-2ce8d280-691b-11e9-84db-4787d7d08436.png)

**Comment by rebeccahum** [comment]
@sebrojas14 Hi! How do you reproduce this?

**Comment by phylaxis** [comment]
I'm seeing a similar issue. In my case what is happening Is I seem to only get a short list of authors, not all the possible authors (users) in my system. At first I thought it was limiting the results to just administrators, but I added a new administrator account and still could not add that new user to a post.

I'm running Wordpress  5.1.1 and Co-Authors Plus 3.3.1

**Comment by rebeccahum** [comment]
@phylaxis Any details on how to reproduce this would be helpful.

**Comment by phylaxis** [comment]
Hey @rebeccahum I'm not really sure as it just started happening, I think with the latest Wordpress update. We also use the 'Members' plug-in and I thought there might be a conflict there, but I tried disabling that plug-in and it didn't make a difference.

The steps to reproduce are really to simply try adding an additional author on a post and see that the list returned is not populated with all the available authors.

**Comment by sebrojas14** [comment]
In my case, install OneSignal Push Notifications plugin and thats all

**Comment by phylaxis** [comment]
I'm circling back on this issue as it still seems to be unresolved. Basically the `coauthors_ajax_suggest` call only seems to return "Admin" users and not any other authors, or any other types of users.

**Comment by phylaxis** [comment]
I actually went back and reviewed the settings for the "Members" plug-in and it looks like this WAS the issue after all (even though I had tried disabling it and still had the issue). One of our admin users had removed the  “Edit Posts” and “Edit Publish Posts” permissions on Authors and that was causing Co-Authors-Plus to not recognize additional authors on the posts (this included "regular" and custom cost types). Sorry for the false alarm.

**Comment by rebeccahum** [comment]
@sebrojas14 can you confirm if this was the case for you?

**Comment by sebrojas14** [comment]
No, It has nothing to do with it. As I said, the problem is with another plugin and is cousing a browser error. Basically, OneSignal plugin throw an js error ONLY when CO-Autors plugin is enabled, causing that co-autors plugon didn't load fully:
![image](https://user-images.githubusercontent.com/1771574/59532271-d8d78480-8ead-11e9-81ff-9d1f4eb1de52.png)
![image](https://user-images.githubusercontent.com/1771574/59532277-de34cf00-8ead-11e9-82c7-7109f87e4686.png)
 So basically, in orther to work I had to disable OneSignal plugin, set authros, enable OneSignal egaind and then publish.
